plugins/providers/bigquery/model.go

```diff
@@ -79,9 +81,10 @@ func (t *Table) FromDomain(r *domain.Resource) error {
 
 func (t *Table) ToDomain() *domain.Resource {
 	r := &domain.Resource{
-		Type: ResourceTypeTable,
-		Name: t.TableID,
-		URN:  fmt.Sprintf("%s:%s.%s", t.ProjectID, t.DatasetID, t.TableID),
+		Type:      ResourceTypeTable,
+		Name:      t.TableID,
+		URN:       fmt.Sprintf("%s:%s.%s", t.ProjectID, t.DatasetID, t.TableID),
+		GlobalURN: fmt.Sprintf("urn:bigquery:%s:%s:%s.%s", t.ProjectID, ResourceTypeTable, t.DatasetID, t.TableID),
```

Comment:
the bigquery urn must be of the format 
`urn:bigquery:project-01:table:project-01:dataset-01.table-01`
```suggestion
		GlobalURN: fmt.Sprintf("urn:bigquery:%s:%s:%s:%s.%s", t.ProjectID, ResourceTypeTable, t.ProjectID, t.DatasetID, t.TableID),
```